docs/auth/README.adoc

```diff
@@ -13,3 +15,136 @@ dependencies {
 
 == Usage
 
+To use the Auth SDK you'll first need to:
+
+* Import the Core module. See link:../core/README.adoc[the Core documentation].
+* Specify the package name of your Android project in `android.defaultConfig.manifestPlaceholders` of the apps `build.gradle` file.
```

Comment:
actually, the value there isn't necessary to be the `package name` of the android project. It is an option, but it really should be the schema of redirect url (and it is recommended to use the package name). See https://github.com/openid/AppAuth-Android#capturing-the-authorization-redirect.

Reply:
Finding this hard to describe properly in a short way, any suggestions on the wording? Like should we go as in depth as https://github.com/openid/AppAuth-Android#capturing-the-authorization-redirect

Reply:
If you want to redirect back to your app, then i think its the only option to use to package name - as this is used in the auth request so Keycloak knows what to redirect back to. If you specify something else that isn't your package name, you will get an exception when trying to build too:

```
Error:Execution failed for task ':auth:processDebugAndroidTestManifest'.
> Manifest merger failed : Attribute data@scheme at manifestMerger8608265398446142713.xml requires a placeholder substitution but no value for <appAuthRedirectScheme> is provided.
```

I think the AppAuth docs are just pointing out that the redirects for mobile are different for the web and you need to use a `com.myapp.example` instead of a regular `https://` url.

Reply:
How about something like this:

Specify the schema of the redirect url for your Android project in `android.defaultConfig.manifestPlaceholders` of the apps `build.gradle` file. 
It should match the schema specified in the "Valid Redirect URL" section for the client in Keycloak. 
It is recommended to use the package name of the Android app as the schema of the redirect  url to avoid conflicts.

Reply:
Thanks @wei-lee @TommyJ1994 !